test/system/docker_live_restore_test.go

```diff
@@ -0,0 +1,171 @@
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+// Define manifests as YAML and then unmarshal it to a Go struct so that it is easier to write and
+// debug, as it can be copy-pasted to a file and applied manually.
+const dockerLiveRestoreOnNodesDSManifest = `apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  generateName: test-live-restore-
+spec:
+  selector:
+    matchLabels:
+      name: test-live-restore
+  template:
+    metadata:
+      labels:
+        name: test-live-restore
+    spec:
+      tolerations:
+      - key: ""
+        operator: Exists
+      terminationGracePeriodSeconds: 1
+      initContainers:
+      - name: test-live-restore
+        image: quay.io/kinvolk/docker:stable
+        imagePullPolicy: IfNotPresent
+        command:
+        - /bin/sh
+        args:
+        - -c
+        - 'docker info -f {{.LiveRestoreEnabled}} | grep true'
+        volumeMounts:
+        - name: docker-socket
+          mountPath: /var/run/docker.sock
+
+      containers:
+      - name: wait
+        image: quay.io/kinvolk/docker:stable
+        imagePullPolicy: IfNotPresent
+        command:
+        - /bin/sh
+        args:
+        - -c
+        - sleep infinity
+      volumes:
+      - name: docker-socket
+        hostPath:
+          path: /var/run/docker.sock
+`
+
+const dockerLogOptsOnNodesDSManifest = `apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  generateName: test-log-opts-
+spec:
+  selector:
+    matchLabels:
+      name: test-log-opts
+  template:
+    metadata:
+      labels:
+        name: test-log-opts
+    spec:
+      tolerations:
+      - key: ""
+        operator: Exists
+      terminationGracePeriodSeconds: 1
+      containers:
+      - name: wait
+        image: quay.io/kinvolk/jq:1.6
+        imagePullPolicy: IfNotPresent
+        command:
+        - /bin/sh
+        args:
+        - -c
+        - sleep infinity
+
+      initContainers:
+      - name: test-log-opts-1
+        image: quay.io/kinvolk/jq:1.6
+        imagePullPolicy: IfNotPresent
+        command:
+        - /bin/sh
+        args:
+        - -c
+        - jq '."log-opts"."max-size"' /etc/docker/daemon.json | grep 100m
+        volumeMounts:
+        - name: docker-config
+          mountPath: /etc/docker/daemon.json
+      - name: test-log-opts-2
+        image: quay.io/kinvolk/jq:1.6
+        imagePullPolicy: IfNotPresent
+        command:
+        - /bin/sh
+        args:
+        - -c
+        - jq '."log-opts"."max-file"' /etc/docker/daemon.json | grep 3
+        volumeMounts:
+        - name: docker-config
+          mountPath: /etc/docker/daemon.json
+      volumes:
+      - name: docker-config
+        hostPath:
+          path: /etc/docker/daemon.json
+`
+
+func TestDockerNodeConfig(t *testing.T) {
+	tcs := []struct {
+		name   string
+		config string
+	}{
+		{"Is_Live_Restore_Enabled", dockerLiveRestoreOnNodesDSManifest},
```

Comment:
We could have the same convention as below.

```suggestion
		{"Docker_Live_Restore_Enabled", dockerLiveRestoreOnNodesDSManifest},
```

Reply:
My bad, will remove the docker from the name in the other test. The invoking test is already called `TestDockerNodeConfig` so not repeating the `Docker` in it.